### ISSUE TITLE
Dragging an image onto the wysiwyg file browser does not work

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18980]
We get an an NPE when trying to render the new dotAsset.

```
17:47:29.588  INFO  reindex.ReindexThread - ReindexEntries found : 1
17:47:29.588  INFO  reindex.ReindexThread - BulkRequests created : 2
17:47:29.826  INFO  log.accessLog - Method execution failed: 
java.lang.NullPointerException: null
	at com.dotmarketing.portlets.browser.ajax.BrowserAjax.listCleanup(BrowserAjax.java:462) ~[classes/:?]
	at com.dotmarketing.portlets.browser.ajax.BrowserAjax.getFolderContentWithDotAssets(BrowserAjax.java:365) ~[classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_162]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_162]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_162]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_162]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule$1.doFilter(CreatorModule.java:229) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule.executeMethod(CreatorModule.java:241) ~[dot.dwr-3rc2modified_3.jar:?]
```

**Comment by wezell** [comment]
PR: #18981

**Comment by dsilvam** [comment]
Passed Internal QA: Dragging an image onto the wysiwyg file browser working.

**Comment by bryanboza** [comment]
fixed, tested on release-5.3.5 // Postgres // FF
